blocks/library/latest-comments/index.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+import { Placeholder, Spinner } from 'components';
+import { __ } from 'i18n';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import './block.scss';
+import { registerBlockType } from '../../api';
+import { getLatestComments } from './data.js';
+import InspectorControls from '../../inspector-controls';
+import TextControl from '../../inspector-controls/text-control';
+import ToggleControl from '../../inspector-controls/toggle-control';
+import BlockDescription from '../../block-description';
+import BlockControls from '../../block-controls';
+import BlockAlignmentToolbar from '../../block-alignment-toolbar';
+
+const MIN_COMMENTS = 1;
+const MAX_COMMENTS = 100;
+
+registerBlockType( 'core/latest-comments', {
+	title: __( 'Latest Comments' ),
+
+	icon: 'list-view',
+
+	category: 'widgets',
+
+	defaultAttributes: {
+		commentsToShow: 5,
+		displayAvatar: true,
+		displayExcerpt: true,
+		displayTimestamp: true,
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( 'left' === align || 'right' === align || 'wide' === align || 'full' === align ) {
+			return { 'data-align': align };
+		}
+	},
+
+	edit: class extends Component {
+		constructor() {
+			super( ...arguments );
+			this.toggleDisplayAvatar = this.toggleDisplayAvatar.bind( this );
+			this.toggleDisplayExcerpt = this.toggleDisplayExcerpt.bind( this );
+			this.toggleDisplayTimestamp = this.toggleDisplayTimestamp.bind( this );
```

Comment:
It would be nice if there was some common abstraction for toggling boolean attributes.